app.html

```diff
@@ -1,22 +1,41 @@
-<!doctype html>
+<!DOCTYPE html>
```

Comment:
why is this in caps?

Reply:
Hmm VS code changes it like that when I save